simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountFragment.kt

```diff
@@ -565,7 +568,8 @@ class AccountFragment : Fragment() {
     val email = this.account.provider.supportEmail
     if (email != null) {
       this.reportIssueGroup.visibility = View.VISIBLE
-      this.reportIssueItem.setOnClickListener {
+      this.reportIssueEmail.text = email.removePrefix("mailto:")
+      this.reportIssueGroup.setOnClickListener {
         val emailIntent =
           Intent(Intent.ACTION_SENDTO, Uri.fromParts("mailto", email, null))
         startActivity(
```

Comment:
It'd be good to wrap this with a call to `resolveActivity` so it doesn't crash on devices lacking an email client:

```kotlin
    if (intent.resolveActivity(packageManager) != null) {
        startActivity(intent)
    }
```

https://developer.android.com/guide/components/intents-common#Email

Reply:
Hm! What we're actually seeing in practice is that devices that lack a configured email app just seem to silently fail here (hence the need to display the email address). I'll wrap it anyway though.

Reply:
Actually, having worked through the chain of method calls, I'm not sure about this one. `Intent.createChooser` can't fail in any way that yields an exception. The `startActivity` call _can_ fail by raising an `ActivityNotFound` exception though...

Reply:
Yeah, it's not essential, this would be pretty rare in practice, but could happen on a managed device like the iPads that were sent out earlier in the year. Just a best practice sort of thing.

You did exactly what I've done in the past, which is just show a dialog in the case of a failure here.